fetch_bringup/config/default_controllers.yaml

```diff
@@ -55,12 +55,19 @@ base_controller:
   # use laser to only slowly collide with things
   laser_safety_dist: 1.0
 
+arm_twist:
+  type: "robot_controllers/CartesianTwistController"
+  root_name: torso_lift_link 
+  tip_name: wrist_roll_link
```

Comment:
this should really be nested under "arm_controller" like the follow joint trajectory